source/content.js

```diff
@@ -74,6 +74,7 @@ import monospaceTextareas from './features/monospace-textareas';
 import improveShortcutHelp from './features/improve-shortcut-help';
 import displayIssueSuggestions from './features/display-issue-suggestions';
 import addPullRequestHotkey from './features/add-pull-request-hotkey';
+import openSelectionInNewTab from './features/add-selection-in-new-tab';
```

Comment:
Is it necessary?

Reply:
That’s part of the merge on master, not this PR